pages/linux/ncat.md

```diff
@@ -1,7 +1,8 @@
 # ncat
 
-> Use the normal `cat` functionality over networks.
-> More information: <https://manned.org/ncat>.
+> Read, write, redirect, and encrypt data across a network.
+> An alternative implementation of a similar utility called netcat/nc.
```

Comment:
```suggestion
> An alternative implementation of a similar utility called `netcat`/`nc`.
```

(or)

```suggestion
> An alternative implementation of a similar utility called Netcat/`nc`.
```

We enclose command names inside backticks. Alternatively for Netcat, we could capitalize the first letter since the project name follows the same.